src/main/java/com/fusionflux/portalcubed/fluids/ToxicGooFluid.java

```diff
@@ -115,7 +115,6 @@ public Block(FlowableFluid flowableFluid, Settings settings) {
         }
 
         @Override
-        @SuppressWarnings("deprecation")
```

Comment:
Why'd you remove this?

Reply:
There was no deprecation warning with it removed

Reply:
There's a deprecation warning in IntelliJ because you're overriding a deprecated method.